### ISSUE TITLE
Is there a document depicting `config_norm` in branch "mlsys-tutorial-2022"? 

### ISSUE THREAD

**Issue by hust-gaoyujing** [ucb-bar/gemmini#241]
I am so interested in the new thing "norm" when i read the codes in branch  "mlsys-tutorial-2022", but i can't find a document for it to understand its function and how to use it. So i want to know whether or not there is a document depicting `config_norm` ?

**Comment by hngenc** [comment]
We're planning to merge that in Gemmini's `master` branch in the next few weeks, along with documentation. We are currently just distracted by some upcoming paper deadlines :)

`config_norm` provides support for LayerNorm operations and [I-BERT's integer-only softmax approximation](https://arxiv.org/pdf/2101.01321.pdf). These are both essentially normalization operations where the innermost dimension is the one being normalized. However, we hope soon to extend `config_norm` to also support operations like BatchNorm, where the normalized dimension is the outermost dimension.

**Comment by hust-gaoyujing** [comment]
Thanks for your reply. Your works are fantastic and instructive to who are instrested in accelerator.Looking forward to more of your  works.

**Comment by hngenc** [comment]
Haha, thank you! :)

**Comment by ybai62868** [comment]
Great work and thanks again! Looking forward to seeing the new slides and records in MLSys-2022-tutorial !
